### PR TITLE
LEARNER-5681

### PR DESCRIPTION
**Pull request by Ayub-Khan**
Increased timeout to fix drive timeout issues.

### CODE REVIEW

**Review by raeeschachar — APPROVED** [review]

**Review by robrap — APPROVED** [review]
We can see if this improves things, but this doesn't seem like the right way to go.  No successful call should be taking over 60 seconds.  The timeout should probably be a lot lower.

Do we know what call is taking this long?  Is it an external call to the payment provider?  We may need to find some way to mock things so that we don't have flakiness tied to an external provider.  Needs some thought.

In other words, I'm not blocking this if you want to see if this helps resolve.  But, I'm planting a seed in case this doesn't succeed and just makes real failures take longer.  We will want to go to a lower timeout and resolve in another way.

Thanks.
